src/Trie.js

```diff
@@ -7,33 +7,20 @@ const getClosestNode = require('./getClosestNode')
 /**
   Trie class
   @name Trie
-  @example const store = new Trie()
+  @example const store = new Trie(...args)
```

Comment:
Bad example. We don't do something like this in a real world

```js
const store = new Trie(...args)
```

Specify something useful instead

```js
const store = new Trie(new Map())
```

Reply:
ok. understood